services/src/floodlight-modules/src/main/java/org/openkilda/floodlight/kafka/RecordHandler.java

```diff
@@ -125,8 +127,8 @@ protected void doControllerMsg(CommandMessage message) {
     }
 
     private void handleCommand(CommandMessage message, String replyToTopic, Destination replyDestination)
-            throws FlowCommandException {
-
+            throws FlowCommandException, SwitchOperationException {
+        logger.info("Request Received {}, {}, {}", message, replyToTopic, replyDestination);
```

Comment:
please remove it, it will break ELK stack again :)
would be better to log specific operation in your doConfigurePort

Reply:
Updated